### ISSUE TITLE
[macOS] Kindle Comic Converter has the version number 0.0.0

### ISSUE THREAD

**Issue by core-code** [ciromattia/kcc#632]
starting with version 5.6.3 and including version 5.6.4 the Kindle Comic Converter claims to have version "0.0.0"

previous versions up to 5.6.2 did not have this problem, so this is a regression. starting with 5.6.3, the in your ```Info.plist``` file the ```CFBundleVersion``` was removed (which is a clear bug and should never happen) and the ```CFBundleShortVersionString``` was set to "0.0.0" (which is just nonsense)

screenshots from the Finder's GetInfo, ColumnView and QuickLook:
<img width="685" alt="Screenshot 2023-11-29 at 19 13 54" src="https://github.com/ciromattia/kcc/assets/1261954/c9de20e4-3a24-4606-8f64-fe10259a7ebb">
<img width="634" alt="Screenshot 2023-11-29 at 19 14 09" src="https://github.com/ciromattia/kcc/assets/1261954/4615afa8-cc71-45bb-be08-1de87dcd118c">
<img width="607" alt="Screenshot 2023-11-29 at 19 13 37" src="https://github.com/ciromattia/kcc/assets/1261954/12cc1bbf-72d3-48eb-8cf8-40948b055e52">

**Comment by core-code** [comment]
<img width="257" alt="Screenshot 2023-11-29 at 19 13 30" src="https://github.com/ciromattia/kcc/assets/1261954/e08a78c1-a8c6-44d6-a285-21e7bd77bff7">

**Comment by axu2** [comment]
Yes, I removed our info.plist since it caused tons of issues. pyinstaller provided defaults values. 

The proper fix will probably involve modifying the pyinstaller command.

**Comment by core-code** [comment]
> The proper fix will probably involve modifying the pyinstaller command.

since the app is unsigned you could also process the Info.plist after the build step e.g. using ```/usr/libexec/PlistBuddy ```

**Comment by axu2** [comment]
The proper way to do this is to switch from building directly from the `.py` to adding the intermediate `.spec` step. https://github.com/pyinstaller/pyinstaller/issues/6943

**Comment by darodi** [comment]
@axu2 

> Also the plist said 5.5.2 which hasn't been updated in a while. Current at the time was 5.6.2
_Originally posted by @axu2 in https://github.com/ciromattia/kcc/issues/632#issuecomment-1832555911_

version was replaced at build time with this code.

https://github.com/ciromattia/kcc/blob/d8957dc4a6ef2941cad6ee1d523a55a5a204c3b1/setup.py#L43-L47

![image](https://github.com/ciromattia/kcc/assets/4682830/eac3771b-851b-413f-bee8-77d28328f0d5)

**Comment by core-code** [comment]
still not fixed in 6.2.0, it still claims to be 0.0.0

**Comment by drshliapa** [comment]
Opened #1385 with a fix. The macOS build now goes through a spec file that
sets CFBundleShortVersionString and CFBundleVersion from the package version,
so the next release should show the real version instead of 0.0.0.
